src/arc.ts

```diff
@@ -77,28 +105,54 @@ export function getWeb3Provider() {
       web3Provider = web3WsProvider;
     }
   }
-  if (web3Provider !== web3WsProvider && web3Provider.networkVersion !== "1512051714758") {
-    console.warn(`YOU ARE NOT CONNECTED TO GANACHE (but to ${web3Provider.networkVersion}) - please switch conection to localhost: 8545 to enable transactions`);
-  } else {
-    console.log(`Connected to Ganache - this is great in this test phase`);
+
+  // if we are connect to metamask, we check if our Arc instance uses the same connection
+  // if not, we ????
+  if (web3Provider && web3Provider.isMetaMask) {
+    console.log(web3Provider);
+    const networkName = Util.networkName(web3Provider.networkVersion);
+    if (networkName === "ganache") {
+      console.log(`Connected to ${networkName} - this is great`);
+    } else if (networkName === "kovan") {
+      console.log(`Connected to ${networkName} - this is not working yet but will soon`);
+    } else {
+      console.warn(`YOU ARE NOT CONNECTED TO GANACHE (but to ${networkName}) - please switch your metamask connection to Kovan or Private network`);
```

Comment:
We can show real notification bubbles for this too using showNotification

Reply:
yeah, but the question is _when_ to show those...